pkg/asset/installconfig/aws/aws.go

```diff
@@ -12,15 +12,19 @@ import (
 	survey "gopkg.in/AlecAivazis/survey.v1"
 
 	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/ipnet"
 	"github.com/openshift/installer/pkg/types/aws"
 )
 
-const (
-	defaultVPCCIDR = "10.0.0.0/16"
+var (
+	defaultVPCCIDR = ipnet.MustParseCIDR("10.0.0.0/16")
 )
 
 var (
-	validAWSRegions = map[string]string{
+	// ValidAWSRegions is a map of the known AWS regions. The key of the map is
+	// the short name of the region. The value of the map is the long name of
+	// the region.
+	ValidAWSRegions = map[string]string{
```

Comment:
This should probably be moved down into types and imported instead of the other way around.